conduct.md

```diff
@@ -120,15 +120,16 @@ TODO: Write an introductory paragraph.
 
 #### Acceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Openness. Students being open will help your team members know what you are capable of doing so they do not assign you something that you would not understand in which doing this could prevent confusion and reserve time.
+- Retaining team members ideas. Students must keep or reserve someones idea to let your teammates know that they are heard within the group and their opinions matter which is a form of respecting everyone's thoughts and opinions.
+- Encourage humility. Students must know that it is important for everyone in a team to understand that it is not about one person but that it is about the whole team because software engineering is a team effort in which everyone needs each other to properly finish a project.
 
 #### Unacceptable Conduct
 
-- TODO
-- TODO
-- TODO
+- Resentment. Students must know that this will disrupt the flow of the team and can lead to miscommunication and/or using inappropriate language which can cause tension increases conflicts.
+- Lack of Communication and miscommunication. Students must know that within a group, communication is key to keep the team on track and one way to practice good communication is by giving good criticism to your teammates.
+i.e When reviewing their code, tell them what they should or should not do but in a respectful way to prevent miscommunication and/or telling them what you like that they did and suggest what you think would be best.
+- Ego. Software engineering is a team effort so having an big ego can disrupt the workflow of the group and leave them feeling less important which is not respectful to others.
```

Comment:
Similar to my comment above, this section should describe what students should not do if a conflict arises. For example, they probably shouldn't try to escalate the conflict or ignore it.